src/commands/getUserDataFolder.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import { fromPromise } from 'rxjs/observable/fromPromise'
+
+import { createCommand, Command } from 'helpers/ipc'
+
+type Input = {}
+
+type Result = string
+
+const cmd: Command<Input, Result> = createCommand('getUserDataFolder', () =>
+  fromPromise(new Promise(resolve => resolve(process.env.LEDGER_CONFIG_DIRECTORY))),
```

Comment:
does it really need to go through command tho?
you can directly do `electron.app.getPath('userData')` from renderer side

it's what is done for the logger https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/helpers/resolveLogsDirectory.js

Reply:
Ah! :joy:  my bad, I first used `import { app } from 'electron'` (which was obviously not working in renderer), but I forgot about `import { remote } from 'electron'` then `remote.app`. Good catch. Fixing.